torch/jit/_script.py

```diff
@@ -44,8 +44,8 @@
 
 type_trace_db = JitTypeTraceStore()  # DB to hold all call traces from MonkeyType
 
-torch._C.ScriptMethod.graph_for = _graph_for  # type: ignore
-torch._C.ScriptFunction.graph_for = _graph_for  # type: ignore
+torch._C.ScriptMethod.graph_for = _graph_for  # type: ignore[attr-defined]
+torch._C.ScriptFunction.graph_for = _graph_for  # type: ignore[attr-defined]
```

Comment:
Can we make another PR for this only? This change is unrelated to the feature being added.